src/extension-utilities.ts

```diff
@@ -79,7 +79,9 @@ export class ExtensionUtilities {
    * @param {boolean} preserveComments
```

Comment:
Can you remove this from the comment block

Reply:
Done. Good catch I missed that.